### PR TITLE
Minor Makefile changes

### PR DESCRIPTION
**Pull request by AlexKornitzer**
Renamed dlang to bin in the Makefile.acr, not sure if this change will be accepted but it makes more sense to me.

Fixed errors in the mdmp.mk file which was causing failures.

### CODE REVIEW

**Comment by AlexKornitzer** [comment]
Oh great that failed -__- Looks like the error is due to `checking pkg-config flags for r_core... no` @Maijin have you any insight on this?

**Comment by AlexKornitzer** [comment]
Ah I may have found the issue

**Comment by Maijin** [comment]
Yes probably you need to update the PKG stuff, also you won't be able to see result until it's pushed (don't relly on travis PR)

File: Makefile.acr [hunk 1]
```diff
@@ -19,8 +19,8 @@ help:
 	@echo "  baleful            - build baleful r2 plugin"
 	@echo "  asm                - build armthumb, ppc, psosvm, x86bea, m68k plugins"
 	@echo "  anal               - build analysis plugins (RAnal)"
+	@echo "  bin                - build binary plugins (RBin)"
 	@echo "  debug              - build debugger plugins (RDebug)"
-	@echo "  dlang              - support DLang demangler (RBin)"
```
Comment (radare) [inline Makefile.acr]: why

**Comment by radare** [comment]
r2pm already sets PKG_CONFIG_PATH to the path that r2 provides.

File: Makefile.acr (hunk 1 above)
Comment (radare) [inline Makefile.acr]: nvm, this target is not used by the dlang pkg

**Comment by radare** [comment]
yep, just checked in radare2-pm, and it’s fine, this target is not used there. sorry for the confusion :)

**Comment by AlexKornitzer** [comment]
Ah yeah sorry that is me misreading the r2pm db. Looks like the error may be due to the MDMP recipe including that line, as other recipes do not seem to have it. Could that be the cause of the r_core error?

**Comment by AlexKornitzer** [comment]
Put the PR in for radare-pm this should fix the error that travis is shouting at us!

**Comment by AlexKornitzer** [comment]
Now that the PR on radare2-pm has been merged can we get travis to re-run?

**Comment by radare** [comment]
Travis will eventyally rerun after a commit in master. We can wait a bit or just manually rerun if you are curious

**Comment by AlexKornitzer** [comment]
Erm I don't mind, whatever is easiest for you. I think sooner rather than later would be preferable as this hasn't actually been installable since the rework! Clearly it is in high demand :P

**Comment by radare** [comment]
should be run already after the pr i merged

**Comment by AlexKornitzer** [comment]
The last run was from a PR 4 hrs ago so I don't think this would have caught the PR on r2pm. Also for the build to work it needs the changes on this PR as well.

**Comment by AlexKornitzer** [comment]
@radare so that has worked, just tested on my Linux build but it fails on my MacOS machine due to an error which I think has been issued on r2 before (cant find it) where brew and other things cause pkg_config errors. Does this ring any bells, if so what was the solution?

```
clang: warning: argument unused during compilation: '-shared'
In file included from bin_mdmp.c:4:
/usr/local/Cellar/radare2/0.10.5/include/libr/r_util.h:54:10: fatal error: 'openssl/bn.h' file not found
#include <openssl/bn.h>
```

**Comment by radare** [comment]
wtf, ssl is not used at all in r2 so just pass HAVE_LIB_SSL=0 in the make step and it will build… but i guess osx openssl version is too old to contain this file or so..

**Comment by AlexKornitzer** [comment]
Haha, my thoughts exactly! This seems to be an issue with r2pm and macos in general, just tried to install pyc and hit the same error. Is this a fix we could apply in r2pm or is it a problem with brew?

**Comment by Maijin** [comment]
I never had issue on OSX with Open ssl stuff

**Comment by AlexKornitzer** [comment]
@Maijin are you brewed?

**Comment by AlexKornitzer** [comment]
Also it seems that r_util.h does rely on openssl: https://github.com/radare/radare2/blob/master/libr/include/r_util.h

**Comment by Maijin** [comment]
Yes but obviously not for r2

**Comment by AlexKornitzer** [comment]
Ah I have it brewed on macos and git on linux

**Comment by AlexKornitzer** [comment]
The link to extras was never updated (just updating the brew formula): The requested URL /get/radare2-extras-1.0.tar.xz was not found on this server.

**Pull request merged**